users/vieting/experiments/switchboard/hybrid/feat/network_helpers/perturbation.py

```diff
@@ -0,0 +1,128 @@
+import functools
```

Comment:
Please remove it from here for now.

Reply:
Removed and commented the relevant parts in experiments.py out.